### ISSUE TITLE
[Problem]: ask for a pair pin

### ISSUE THREAD

**Issue by Enjoy-Combi** [mikebrady/shairport-sync#1672]
### What happened?

When using the iPad Pro (iOS16), I get the request for the code on the screen. If I look at the log, here what I get :
         1.498161777 "shairport.c:254" "soxr" interpolation has been chosen.
         5.028996367 "rtsp.c:2568" Connection 1: Unhandled POST /pair-pin-start Content-Length 0
         0.006025778 "rtsp.c:1353" Connection 1: Connection closed by client.

Under iPhone with iOS16, no issue.

### Relevant log output

```shell
0.126910611 "shairport.c:2200" Startup in AirPlay 2 mode, with features 0x405c4a00,0x1c340 on device "dc:a6:32:57:03:17".
         0.000133000 "shairport.c:2239" Version String: "4.1.1-5-g69d3b02a-AirPlay2-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc"
         0.000022833 "shairport.c:2258" Command Line: "shairport-sync -v".
         0.000075926 "shairport.c:2265" libsodium initialised.
         0.000438944 "shairport.c:2290" libgcrypt initialised.
         0.000056704 "shairport.c:2294" Log Verbosity is 1.
         0.000047315 "audio_alsa.c:1031" alsa: alsa_maximum_stall_time of 0.200000 sec.
         0.000042630 "audio_alsa.c:1320" alsa: disable_standby_mode is "never".
         0.000015722 "audio_alsa.c:1324" alsa: disable_standby_mode_silence_threshold is 0.040000 seconds.
         0.000015759 "audio_alsa.c:1326" alsa: disable_standby_mode_silence_scan_interval is 0.004000 seconds.
         0.000036278 "audio_alsa.c:1366" alsa: output device name is "default".
         0.000234111 "shairport.c:2342" disable_resend_requests is off.
         0.000044148 "shairport.c:2343" diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
         0.000017704 "shairport.c:2347" statistics_requester status is 0.
         0.000014833 "shairport.c:2352" rtsp listening port is 7000.
         0.000014834 "shairport.c:2353" udp base port is 6001.
         0.000014351 "shairport.c:2354" udp port range is 10.
         0.000014334 "shairport.c:2355" player name is "Enjoy-combi".
         0.000014166 "shairport.c:2356" backend is "alsa".
         0.000014426 "shairport.c:2357" run_this_before_play_begins action is "(null)".
         0.000016241 "shairport.c:2358" run_this_after_play_ends action is "(null)".
         0.000015519 "shairport.c:2359" wait-cmd status is 0.
         0.000014333 "shairport.c:2360" run_this_before_play_begins may return output is 0.
         0.000014407 "shairport.c:2361" run_this_if_an_unfixable_error_is_detected action is "(null)".
         0.000014500 "shairport.c:2363" run_this_before_entering_active_state action is  "(null)".
         0.000014741 "shairport.c:2365" run_this_after_exiting_active_state action is  "(null)".
         0.000014870 "shairport.c:2367" active_state_timeout is  10.000000 seconds.
         0.000017519 "shairport.c:2368" mdns backend "(null)".
         0.000014481 "shairport.c:2370" interpolation setting is "auto".
         0.000014519 "shairport.c:2374" interpolation soxr_delay_threshold is 30000000.
         0.000014648 "shairport.c:2375" resync time is 0.050000 seconds.
         0.000014870 "shairport.c:2376" allow a session to be interrupted: 0.
         0.000014112 "shairport.c:2377" busy timeout time is 0.
         0.000014963 "shairport.c:2378" drift tolerance is 0.001995 seconds.
         0.000015185 "shairport.c:2379" password is "(null)".
         0.000014277 "shairport.c:2380" ignore_volume_control is 0.
         0.000014297 "shairport.c:2384" volume_max_db is not set
         0.000014407 "shairport.c:2385" volume range in dB (zero means use the range specified by the mixer): 0.
         0.000022519 "shairport.c:2387" volume_range_combined_hardware_priority (1 means hardware mixer attenuation is used first) is 0.
         0.000022018 "shairport.c:2391" playback_mode is 0 (0-stereo, 1-mono, 1-reverse_stereo, 2-both_left, 3-both_right).
         0.000016648 "shairport.c:2393" disable_synchronization is 0.
         0.000014111 "shairport.c:2394" use_mmap_if_available is 1.
         0.000014241 "shairport.c:2395" output_format automatic selection is enabled.
         0.000014574 "shairport.c:2399" output_rate automatic selection is enabled.
         0.000014611 "shairport.c:2403" audio backend desired buffer length is 0.200000 seconds.
         0.000015556 "shairport.c:2405" audio_backend_buffer_interpolation_threshold_in_seconds is 0.120000 seconds.
         0.000015481 "shairport.c:2407" audio backend latency offset is 0.000000 seconds.
         0.000015260 "shairport.c:2409" audio backend silence lead-in time is "auto".
         0.000014389 "shairport.c:2413" zeroconf regtype is "_raop._tcp".
         0.000014796 "shairport.c:2414" decoders_supported field is 1.
         0.000014333 "shairport.c:2415" use_apple_decoder is 0.
         0.000014019 "shairport.c:2416" alsa_use_hardware_mute is 0.
         0.000020481 "shairport.c:2420" no special mdns service interface was requested.
         0.000053000 "shairport.c:2423" configuration file name "/etc/shairport-sync.conf" resolves to "/etc/shairport-sync.conf".
         0.000024611 "shairport.c:2453" loudness is 0.
         0.000016241 "shairport.c:2454" loudness reference level is -20.000000
         0.000230426 "shairport.c:2464" size of hw_addr is 8.
         0.005700111 "shairport.c:2547" NQPTP is online.
         1.498161777 "shairport.c:254" "soxr" interpolation has been chosen.
         5.028996367 "rtsp.c:2568" Connection 1: Unhandled POST /pair-pin-start Content-Length 0
         0.006025778 "rtsp.c:1353" Connection 1: Connection closed by client.
```

### Configuration Information.

>> Display Config Start.

From "uname -a":
 Linux raspberry 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux

From /etc/os-release:
 Debian GNU/Linux 11 (bullseye)

From /sys/firmware/devicetree/base/model:
 Raspberry Pi 4 Model B Rev 1.2

Shairport Sync Version String:
 4.1.1-5-g69d3b02a-AirPlay2-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc

Command Line:
 shairport-sync --displayConfig

Configuration File:
 /etc/shairport-sync.conf

The Configuration file contains no active settings.

>> Display Config End.
>> Goodbye!

### How did you install Shairport Sync?

Built from source

### Check previous issues

- [X] Confirm

**Comment by mikebrady** [comment]
Thanks for the post. This is a mystery. Is there anything special about the configuration of the iPad? Any security settings, VPNs, etc?

**Comment by Enjoy-Combi** [comment]
Hello,

Yesterday I bought a sonos Era 100 and same issue. Probably something wrong with the iPad. Will try with another one.
It might be related to the replacement of the screen, not orignal. Since I have done it, Faceid not working and probably some issues with security.

I keep you in touch

**Comment by slynn1324** [comment]
I ran into the same thing from my iPhone on iOS 16.6.1 (updating now to see if that fixes it) - whenever I tried to connect my phone would prompt me for the pin, and the Shairport-Sync logs would show "Unhandled POST /pair-pin-start".  In digging a little further, I came across a description of the pairing process here -> https://github.com/ejurgensen/pair_ap, that indicates that for some devices a transient pairing process is used and the fixed code "3939" is used.  On a whim, I tried the code 3939 and it worked.  (Other codes, 1234, 0000, etc did not).  Once paired the first time, I'm not re-prompted for the pin again.
